### DISCUSSION TITLE
Relacionamento N para N

### DISCUSSION THREAD

**Discussion by dhionys-soares** [balta-io/forum#466] [Tira Dúvidas] [answered]
Carreira: Data Access, Curso: Fundamentos do SQL Server, aula: Composite Key.

Dúvida: Então para criar um relacionamento de muitos para muitos em SQL, é preciso criar uma tabela que contenha os id's das entidades relacionadas(aluno e curso) dentro de um constraint de PK_Progresso e ambos os id's dever ser PK. É isso?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @dhionyssoares, como vai? Isto mesmo, isto é chamado de tabela associativa e ela carrega os IDs das duas tabelas que ela "associa" ou relaciona.
